### PR TITLE
update ircd docs for inspircd migration

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: docs/services/irc.md [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
-OPME is no fun if you can't use it to take over any channel you want, patch
-`contrib/m_opme.c` before compiling to make the function `chan_is_opless` always
-`return 1`:
+#### Important Configuration
 
-```c
-chan_is_opless(const struct Channel *const chptr) {
-  const dlink_node *ptr = NULL;
+_oper_ is a list of admin users on the irc server. Their OPER password will need
+to be manually hashes with hmac-sha256 and placed in a secret on the server to
```
Comment (cmgn) [inline docs/services/irc.md]: s/hashes/hashed

File: docs/services/bitlbee.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-default bitlbee settings. Its settings can be found in
-`/etc/docker-compose/services/bitlbee`, see
-[docker-sevices repo](https://github.com/redbrickCmt/docker-compose-services)
-for configs.
+bitlbee is run using the nixos bitlbee service, see
+[here](https://search.nixos.org/options?query=services.bitlbee) for config
+options.
 
-### Update
+We use default bitlbee settings. Its settings can be found in
```
Comment (cmgn) [inline docs/services/bitlbee.md]: s/default/the default

File: docs/services/bitlbee.md [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+[nixos repo](https://github.com/redbrick/nix-configs/blob/master/services/bitlbee.nix).
 
-To update the easiest way is to rebuild the container. Run
-`docker-compose build --no-cache` and then `docker-compose up -d`
+`bitlbee.redbrick.dcu.ie` is not expose externally. It is available at
```
Comment (cmgn) [inline docs/services/bitlbee.md]: s/expose/exposed

File: docs/services/irc.md [hunk 4]
```diff
@@ -1,83 +1,83 @@
...
-should NEVER be automatically upgraded. Letting apt upgrade hybrid will cause us
-to fall off the network, and things to asplode.
+We run Inspircd v3 on Metharme. InspIRCd's docs can be found
+[here](https://docs.inspircd.org/) for configuration specifics.
 
-Most of the stuff in ircd.conf is fairly obvious. For adding new external users
-etc. just copy and change one of the existing blocks. After changes are made to
-the file /rehash as an oper, don't restart it with the init script.
+IRC is available at `irc.redbrick.dcu.ie` on port 6697. SSL is required for
+connection we do not support non-SSL.
```
Comment (cmgn) [inline docs/services/irc.md]: s/connection/connection,

File: docs/services/irc.md [hunk 5]
```diff
@@ -1,83 +1,83 @@
...
-`contrib/m_opme.c` before compiling to make the function `chan_is_opless` always
-`return 1`:
+#### Important Configuration
 
-```c
-chan_is_opless(const struct Channel *const chptr) {
-  const dlink_node *ptr = NULL;
+_oper_ is a list of admin users on the irc server. Their OPER password will need
+to be manually hashes with hmac-sha256 and placed in a secret on the server to
+be read in and placed in a secret on the server to be read in.
```
Comment (cmgn) [inline docs/services/irc.md]: Same thing on both sides of the "and"

File: docs/services/irc.md [hunk 6]
```diff
@@ -1,83 +1,83 @@
...
-See
-[docker-sevices repo](https://github.com/redbrickCmt/docker-compose-services)
-for configs.
+We run a [bridge](https://github.com/qaisjp/go-discord-irc) between Redbrick
+discord and irc. The configuration for this is
+[here](https://github.com/redbrick/nix-configs/tree/master/services/ircd/discord/conf.nix).
 
-To build and deploy a version of irc from source run
-`docker-compose up -d --build`
+The bridge adds all users to from discord with the suffix `_d2` and all irc
```
Comment (cmgn) [inline docs/services/irc.md]: s/to from discord/from discord

**Review by m1cr0man — DISMISSED** [review]
Yeah this is all great. Nothing to add beyond cmgn's requests.

File: docs/services/irc.md (hunk 5 above)
Comment (butlerx) [reply docs/services/irc.md]: im just a record skipping

**Comment by butlerx** [comment]
typos and gramatical  mistakes fixed

**Review by m1cr0man — APPROVED** [review]

**Pull request merged**